### ISSUE TITLE
How long did it take for you to train the five task model

### ISSUE THREAD

**Issue by aodwals** [TxpHome/DFPIR#9]
How long did it take for you to train the five task model？
If training based on all the data in data-dir, would it take a long time to use only one 3090?
deblur 2103 * 30=63090
derain 200 * 80=16000
delol  485 * 60=29100
dehaze  72135 * 1=72135
denoise 5144 * 3=15432
total=195757
And what are the images in the test set?

**Comment by aodwals** [comment]
Thanks for your  code and work very much ,dear author.

**Comment by TxpHome** [comment]
By leveraging a pre-trained model from a three-degradation task and fine-tuning it on a five-degradation task, the total training time is substantially reduced, requiring only about ten days.
